### ISSUE TITLE
logo

### ISSUE THREAD

**Issue by not7cd** [not7cd/crypt#1]
Simple SVG

**Comment by gurjarshakti** [comment]
hey @not7cd  need a logo for this project ?

**Comment by not7cd** [comment]
@gurjarshakti, yes. I'm open for suggestions. I'm happy with what I found [here](https://www.flaticon.com/free-icon/crypt-key_94483#term=crypt&page=1&position=4), but some work needs to be done. Haven't write more about it since it's my todo.

**Comment by gurjarshakti** [comment]
Ok , I can done the work.. explain me a Little bit ...What changes you need with that crypt icon.
